src/main/java/eu/xlime/kafka/rdf/Summary.java

```diff
@@ -0,0 +1,53 @@
+package eu.xlime.kafka.rdf;
+
+import java.util.Map;
+import java.util.Date;
+import java.util.Calendar;
+
+public class Summary {
+	private String consumerId;
+	private Map<String,Long> annotations;
+	private long processed;
```

Comment:
maybe it's better to simply allow 'clients' of this class to add a custom 'messagesProcessed' counter to the 'annotations' map.